api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TxCurrCohortQueries.java

```diff
@@ -1618,7 +1618,7 @@ public SqlCohortDefinition getPatientsWithSemiAnnualTypeOfDispensation() {
             + "                 INNER JOIN obs o ON e.encounter_id = o.encounter_id"
             + "                 WHERE"
             + "                     e.encounter_type = ${6}"
-            + "                         AND (o.concept_id= 23730 OR o.concept_id = 23739 OR o.concept_id = 23888)"
+            + "                         AND (o.concept_id= ${23730} OR o.concept_id = ${23739} OR o.concept_id = ${23888})"
```

Comment:
what if you would have done AND concept_id IN(${23730}, ${23739},${23888}) instead?